kpi/management/commands/populate_assetversions.py

```diff
@@ -112,18 +112,21 @@ def _replace_deployment_ids(_AssetVersion, _Asset):
         for a_id in a_ids:
             # `only()` saves about 8/1000s per iteration
             asset = _Asset.objects.only('_deployment_data').get(id=a_id)
-            version_id = asset._deployment_data['version']
+            version_id = asset.deployment.version_id
             if isinstance(version_id, int):
                 try:
                     # `only()` saves about 5/1000s per iteration
                     uid = asset.asset_versions.only('uid').get(
                         _reversion_version_id=version_id).uid
-                    if 'version_uid' not in asset._deployment_data or \
-                            asset._deployment_data['version_uid'] != uid:
-                        asset._deployment_data['version_uid'] = uid
+                    version_uid = asset.deployment.get_data('version_uid')
+                    if (
+                        version_uid is None  # noqa
+                        or version_uid != uid  # noqa
```

Comment:
what is the linter complaining about here? it looks perfectly acceptable to me.

Reply:
Removed